tests/system/small/test_series.py

```diff
@@ -273,21 +273,31 @@ def test_series_replace_list_scalar(scalars_dfs):
 
 
 @pytest.mark.parametrize(
-    ("values",),
+    ("method",),
     (
-        ([None, 1, 2, None, None, 16, None],),
-        ([None, None, 3.6, None],),
-        ([403.2, None, 352.1, None, None, 111.9],),
+        ("linear",),
+        ("values",),
+        ("slinear",),
+        ("nearest",),
+        ("zero",),
+        ("pad",),
     ),
 )
-def test_series_interpolate(values):
-    pd_series = pd.Series(values)
+def test_series_interpolate(method):
+    values = [None, 1, 2, None, None, 16, None]
+    index = [-3.2, 11.4, 3.56, 4, 4.32, 5.55, 76.8]
+    pd_series = pd.Series(values, index)
     bf_series = series.Series(pd_series)
 
     # Pandas can only interpolate on "float64" columns
     # https://github.com/pandas-dev/pandas/issues/40252
-    pd_result = pd_series.astype("float64").interpolate()
-    bf_result = bf_series.interpolate().to_pandas()
+    pd_result = pd_series.astype("float64").interpolate(method=method)
+    bf_result = bf_series.interpolate(method=method).to_pandas()
+
+    print("pandas")
+    print(pd_result.to_string())
+    print("bigframes")
+    print(bf_result.to_string())
```

Comment:
More leftover debugging code. FYI: `pytest ... --pdb` is really useful for this sort of thing. You can go up and down the stack when an exception is thrown and inspect anything.

```suggestion
```

Reply:
removed